src/pybind/mgr/dashboard/frontend/src/app/core/navigation/identity/identity.component.html

```diff
@@ -0,0 +1,20 @@
+<div dropdown>
+  <a dropdownToggle class="dropdown-toggle" data-toggle="dropdown" i18n-title title="Logged in user">
+    <i class="fa fa-user fa-fw"></i>
+    <span class="caret"></span>
+  </a>
+  <ul *dropdownMenu class="dropdown-menu dropdown-menu-right" role="menu">
+    <li>
```

Comment:
I suggest to use ``<li class="disabled">`` to disable the selection of the menu item. See https://getbootstrap.com/docs/3.3/components/#dropdowns-disabled.